thirdeye/thirdeye-frontend/app/pods/custom/dimensions-table/change-bars/template.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="rootcause-dimensions-table__bar-container {{collapseClassNegative}} {{expandClassNegative}}">
+  <div class="rootcause-dimensions-table__bar rootcause-dimensions-table__bar--negative" style="width:{{record.elementWidth.negative}}"></div>
```

Comment:
Is there a reason for the inline styles here?

Reply:
Specific widths are needed for each bar after row values are scaled. This is similar to the way we would have to do it in D3. Let me know if you have any ideas.

var svg = d3.select("#chartholder")
			.style('width', width + 'px')
			.style('height', height + 'px');